erpnext_documentation/www/docs/user/manual/en/using-erpnext/video.md

```diff
@@ -7,7 +7,7 @@ To access Videos, go to:
 
```

Comment:
definition in bold, 
add 1-2 lines description summary on what this doctype is about